src/main/java/io/wispforest/lavender/book/LavenderClientStorage.java

```diff
@@ -26,6 +35,9 @@ public class LavenderClientStorage {
     private static final TypeToken<Map<UUID, Map<Identifier, Set<Identifier>>>> VIEWED_ENTRIES_TYPE = new TypeToken<>() {};
     private static Map<UUID, Map<Identifier, Set<Identifier>>> viewedEntries;
 
+    private static final TypeToken<Map<UUID, List<ActiveStructureOverlay>>> ACTIVE_STRUCTURES_TYPE = new TypeToken<>() {};
```

Comment:
I do not believe persisting the active structures is in any way necessary, I believe the amount of times this would come in useful does not justify having the code for it around.

All in all, this feature should be removed

Reply:
honestly I would find it rather useful

like, you might be building something across multiple playtime sessions

eg. if you have a big structure you need to build, you'll build part of it in session 1, then exit the game. then when you start session 2 you resume building it and finish it

Reply:
Moving this commit to a separate PR.